services/payments/tests/unit/test_services_payments__get_invoice.py

```diff
@@ -0,0 +1,109 @@
+# pylint: disable=protected-access
+# pylint: disable=redefined-outer-name
+# pylint: disable=too-many-arguments
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+from collections.abc import Iterator
+from datetime import datetime, timedelta, timezone
+from decimal import Decimal
+from typing import cast
+
+import pytest
+import sqlalchemy as sa
+from fastapi import FastAPI
+from models_library.api_schemas_webserver.wallets import PaymentID
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from pydantic import HttpUrl
+from pytest_simcore.helpers.rawdata_fakers import random_payment_transaction
+from pytest_simcore.helpers.typing_env import EnvVarsDict
+from pytest_simcore.helpers.utils_envs import setenvs_from_dict
+from respx import MockRouter
+from simcore_postgres_database.models.payments_transactions import (
+    PaymentTransactionState,
+    payments_transactions,
+)
+from simcore_service_payments.db.payments_transactions_repo import (
+    PaymentsTransactionsRepo,
+)
+from simcore_service_payments.services import payments
+from simcore_service_payments.services.stripe import StripeApi
+
+pytest_simcore_core_services_selection = [
+    "postgres",
+]
+pytest_simcore_ops_services_selection = [
+    "adminer",
+]
+
+
+@pytest.fixture
+def app_environment(
+    monkeypatch: pytest.MonkeyPatch,
+    app_environment: EnvVarsDict,
+    with_disabled_rabbitmq_and_rpc: None,
+    postgres_env_vars_dict: EnvVarsDict,
+    wait_for_postgres_ready_and_db_migrated: None,
+):
+    # set environs
+    monkeypatch.delenv("PAYMENTS_RABBITMQ", raising=False)
+    monkeypatch.delenv("PAYMENTS_POSTGRES", raising=False)
+
+    return setenvs_from_dict(
+        monkeypatch,
+        {
+            **app_environment,
+            **postgres_env_vars_dict,
+            "POSTGRES_CLIENT_NAME": "payments-service-pg-client",
+        },
+    )
+
+
+@pytest.fixture()
+def populate_payment_transaction_db(
```

Comment:
Please let's keep consistency in the fixtures naming as much as possible. 

Please double check the approach we follow in payments service. For instance see how `pytest_simcore.faker_payments_data` fixtures get overriden and extended on `payments` tests.

Reply:
Yes, I am using those: for example wallet_id and user_id. That is sufficient for me for this test :+1: Regarding naming, I am keeping consistency and using the same as is in auto_recharge_listener test, do you have some preferred naming?